lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -177,6 +223,60 @@ WebProcessingServiceCatalogFunction.parameterConverters = [
         }
     },
     {
+         id: 'RegionGeometry',
+         inputToFunctionParameter: function(catalogFunction, input) {
+             if (!defined(input.ComplexData) || !defined(input.ComplexData.Default) || !defined(input.ComplexData.Default.Format) || !defined(input.ComplexData.Default.Format.Schema)) {
+                 return undefined;
+             }
+
+             var schema = input.ComplexData.Default.Format.Schema;
+             if (schema.indexOf('http://geojson.org/geojson-spec.html#') !== 0) {
+                 return undefined;
+             }
+
+             if (schema.substring(schema.lastIndexOf('#') + 1) !== 'multipolygon') {
+                 return undefined;
+             }
+
+             if (input.Identifier.indexOf('region') === -1) {
```

Comment:
I think this test is pretty suspect, too.

It seems to me, anytime a WPS wants a polygon, the UI should allow the user to specify that polygon in a number of ways, including:
* By clicking out polygon points.
* By drawing a bounding box.
* By selecting a region.
* Maybe: by selecting an existing polygon on the map?

It's up to the user how they define their polygon, it's not necessarily specified by the server (though perhaps there could be a mechanism for the server to hint at a good default).

Is that totally crazy?